src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -65,10 +65,14 @@ class Container implements ResettableContainerInterface
 
     protected $services = array();
     protected $methodMap = array();
-    protected $privates = array();
     protected $aliases = array();
     protected $loading = array();
 
+    /**
+     * @deprecated since 3.2, to be removed in 4.0
+     */
+    protected $privates = array();
```

Comment:
Note that, currently, removing this in 4.x brings back the original issue. IMO we should cover 4.x behavior if we deprecate this (ie. why i went with randomzing, it makes `$this->privates` truly a bridge till 4.x).